src/main/webapp/app/instructor-exercise-dashboard/instructor-exercise-dashboard.component.ts

```diff
@@ -37,7 +38,20 @@ export class InstructorExerciseDashboardComponent implements OnInit {
     back() {
         this.router.navigate([`/course/${this.courseId}/instructor-dashboard`]);
     }
+    public setStatistics() {
```

Comment:
Should this really be public for the sake of being able to test it? If possible, I would always try to test the public methods and functionalities of components/classes, which in turn then call private methods like this one

Reply:
the `setStatistics` is only used in `loadExercise`, which is also private. The other reason why I made it public is that `setStatistics` only accesses public member variables.